app/models/category.rb

```diff
@@ -1,3 +1,14 @@
 class Category < ActiveRecord::Base
-  has_many :videos
+  has_many :videos, -> { order("created_at DESC")}
+
+
+  def recent_videos
+    videos.first(6)
+  end
+  
+  #  def recent_videos
+  #   @category = Video.take(6)
+  #   @category.sort_by{ |a| a.created_at }.reverse
+  # end
+
```

Comment:
Previous attempts should be deleted, or captured in your git commits rather than your actual code. Its disturbing to read commented areas like this one for anyone who reviews.